gift-certificate-application/controller/src/main/java/com/epam/esm/controller/exception/AppExceptionHandler.java

```diff
@@ -0,0 +1,150 @@
+package com.epam.esm.controller.exception;
+
+import com.epam.esm.service.exception.*;
```

Comment:
Avoid such "wildcard" imports, please read here why:
https://tharakamd-12.medium.com/is-it-bad-to-use-wildcard-imports-in-java-1b46a863b2be